test/ecs/serialization.spec.ts

```diff
@@ -229,11 +229,11 @@ describe('Serialization Types', () => {
       Default = 0,
       Red = 2,
       Green = 0x33,
-      Pink = 0xff290323
+      Pink = 0x1f290323
```

Comment:
0xffffffff didn't work?

Reply:
nope, typescript hex representation isn't a 32bit one, so numbers > 0x7fff_ffff are out of range

Reply:
But `-1` is a valid number :)